doc/script_commands.txt

```diff
@@ -3582,6 +3582,32 @@ This command does not count skills which are set as flag 4 (permament granted) (
 
 ---------------------------------------
 
+*getrandmobid(<type>{,<flag>{,<level>}})
+
+This command return a random monster ID from the random monster group.
+If <level> is provided it will check against the monster level.
+It will return 0 if invalid.
+
+Valid <type> are:
+	MOBG_BRANCH_OF_DEAD_TREE    = 0 // (default)
+	MOBG_PORING_BOX             = 1
+	MOBG_BLOODY_DEAD_BRANCH     = 2
+	MOBG_RED_POUCH_OF_SURPRISE  = 3
+	MOBG_CLASSCHANGE            = 4
+	MOBG_TAEKWON_MISSION        = 5
+	
+Valid <flag> are:
+	RMF_NONE            = 0x00, // Apply no flags
+	RMF_DB_RATE         = 0x01, // Apply the summon success chance found in the list (otherwise get any monster from the db)
+	RMF_CHECK_MOB_LV    = 0x02, // Apply a monster level check
+	RMF_MOB_NOT_BOSS    = 0x04, // Selected monster should not be a Boss type (default)
+	                            // (except those from MOBG_BLOODY_DEAD_BRANCH)
+	RMF_MOB_NOT_SPAWN   = 0x08, // Selected monster must have normal spawn
+	RMF_MOB_NOT_PLANT   = 0x10, // Selected monster should not be a Plant type
+	RMF_ALL             = 0xFF, // Apply all flags
```

Comment:
`RMF_*` constants must be exported for the script engine via `script_constants.hpp`